packages/api/app/Controllers/Http/ReviewerController.js

```diff
@@ -0,0 +1,211 @@
+	}
+
+	async show({ request }) {
+		return Reviewer.query().withParams(request);
+	}
+
+	async store({ auth, request }) {
+		const { categories } = request.all();
+		const user = await auth.getUser();
+
+		let reviewer;
+		let trx;
+		try {
+			const { init, commit } = getTransaction();
+			trx = await init();
+			reviewer = await Reviewer.create({}, trx);
+			await reviewer.user().associate(user, trx);
+			await this.syncronizeCategories(trx, categories, reviewer);
+
+			await commit();
+		} catch (error) {
+			trx.rollback();
+			throw error;
+		}
+		await reviewer.loadMany(['user', 'categories']);
+		return reviewer;
+	}
+
+	async update({ auth, request }) {
+		const { categories } = request.all();
+		const user = await auth.getUser();
+
+		let reviewer;
+		let trx;
+		try {
+			const { init, commit } = getTransaction();
+			trx = await init();
+			reviewer = await Reviewer.getReviewer(user);
+			if (categories) {
+				await this.syncronizeCategories(trx, categories, reviewer, true);
+			}
+			await commit();
+		} catch (error) {
+			trx.rollback();
+			throw error;
+		}
+		await reviewer.loadMany(['user', 'categories']);
+		Bull.add(Job.key, null);
+		return reviewer;
+	}
+
+	async updateReviewerStatus({ params, request }) {
+		const reviewer = await Reviewer.findOrFail(params.id);
+		const { status } = request.only(['status']);
+		reviewer.merge({ status });
+		await reviewer.save();
+		if (status === 'approved') {
+			const userReviewer = await User.findOrFail(reviewer.user_id);
+			const reviewerRole = await Role.getRole(roles.REVIEWER);
+			await userReviewer.role().dissociate();
+			await userReviewer.role().associate(reviewerRole);
+			await this.sendEmailApprovedReviewer(userReviewer);
+			Bull.add(Job.key, null);
+		}
+
+		return reviewer;
+	}
+
+	async makeRevision({ auth, request, response }) {
+		const { technology } = request.params;
+		const technologyInst = await Technology.getTechnology(technology);
+		if (!['in_review', 'requested_changes', 'changes_made'].includes(technologyInst.status)) {
+			return response.status(400).send(
+				errorPayload(
+					errors.STATUS_NO_ALLOWED_FOR_REVIEW,
+					request.antl('error.reviewer.statusNoAllowedForReview', {
+						status: technologyInst.status,
+					}),
+				),
+			);
+		}
+
+		const user = await auth.getUser();
+		const reviewer = await Reviewer.getReviewer(user);
+		const data = request.only(['description', 'assessment']);
+
+		let trx;
+		let revision;
+		try {
+			const { init, commit } = getTransaction();
+			trx = await init();
+
+			revision = await reviewer.revisions().create(
+				{
+					description: data.description ? data.description : null,
+					assessment: data.assessment,
+				},
+				trx,
+			);
+			await revision.technology().associate(technologyInst, trx);
+			await revision.reviewer().associate(reviewer, trx);
+			technologyInst.status = revision.assessment;
+			await technologyInst.save(trx);
+			await revision.loadMany(['technology', 'reviewer.user']);
```

Comment:
```suggestion
			return revision.loadMany(['technology', 'reviewer.user']);
```

qual utilidade desse loadMany? não estou vendo esse valor ser retornado